test/e2e/gardener/shoot/create_hibernate_wakeup_delete_workerless.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shoot
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	e2e "github.com/gardener/gardener/test/e2e/gardener"
+)
+
+var _ = Describe("Shoot Tests", Label("Shoot", "default", "workerless"), func() {
```

Comment:
Adapt the condition checks in `CreateShootAndWaitForCreation`.
https://github.com/gardener/gardener/blob/5abed1bfedd6f2188e5dcec9af161d1b5c286236/test/framework/k8s_utils.go#L324-L330